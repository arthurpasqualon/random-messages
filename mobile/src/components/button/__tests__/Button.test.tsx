import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../index';

test('renders correctly', () => {
  const tree = renderer.create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});
