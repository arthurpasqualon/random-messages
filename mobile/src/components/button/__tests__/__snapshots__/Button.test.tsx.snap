// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#800080",
      "borderColor": "transparent",
      "borderWidth": 1,
      "opacity": 1,
      "paddingBottom": 10,
      "paddingLeft": 10,
      "paddingRight": 10,
      "paddingTop": 10,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontFamily": "RobotoCondensed-Regular",
          "fontSize": 14,
          "textAlign": "center",
        },
      ]
    }
  />
</View>
`;
